### DISCUSSION TITLE
2012-6 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#7] [Announcements]
# 2012-6 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch6/2012-6.html

**Comment by Timothy-Liuxf** [comment]
这题似乎是少了任取 $\{ x_n \}$ 有界的条件？如果是的话，
1. 取 $x_k = \varepsilon_k$，得到 $\lVert f_n \rVert \geq \sum\limits_k \lVert y_k \rVert$；取 $x_n$ 的范数为 $\max\limits_k x_k$，得到 $\lVert f_n \rVert \leq \sum\limits_k \lVert y_k \rVert$
2. 一致有界性原理
